docs/reference/search-application/apis/put-search-application.asciidoc

```diff
@@ -49,6 +49,20 @@ The <<search-template,search template>> associated with this search application.
 - The template may be modified with subsequent <<put-search-application,put search application>> requests.
 - If no template is specified when creating a search application, or if a template is removed from a search application, we use the <<query-string-query-ex-request,query_string>> defined in the template examples as a default.
 - This template will be used by the <<search-application-search,search application search>> API to execute searches.
+- The template accepts an optional `dictionary` parameter which defines a JSON schema used for validating parameters sent to the <<search-application-search,search application search>> API
+
+.Properties of `<template>`
+[%collapsible%open]
+=====
+
+`script`::
+(Required, object)
+The associated mustache template.
```

Comment:
Should `mustache` be a link to the Mustache docs or the docs that @saarikabhasi wrote for mustache examples?

Reply:
We already link to the search template examples in:

https://github.com/elastic/elasticsearch/blob/4ea79d6bbd4cf4c5d01089b282f1e5848ae96dfb/docs/reference/search-application/apis/put-search-application.asciidoc?plain=1#L45

I don't think it is necessary to link to the search template examples every for every mention of templates.